dhis-2/dhis-api/src/main/java/org/hisp/dhis/eventvisualization/SimpleDimensionHandler.java

```diff
@@ -139,9 +139,7 @@ private List<BaseDimensionalItemObject> loadDimensionalItems( final String dimen
     private SimpleDimension createSimpleEventDimensionFor( final DimensionalObject dimensionalObject,
         final Attribute attribute )
     {
-        final SimpleDimension simpleDimension = new SimpleDimension();
```

Comment:
thank you very much 🎉

Reply:
annoying that simple things often lead to more problems 😬 constructors seem to have gone out of fashion at some point 😂